packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -59,6 +59,12 @@ registerStyles(
       transition: color 0.2s;
     }
 
+    [part='input-field'] ::slotted(:is(input, textarea)) {
```

Comment:
**The Lumo Theme:**
```
[part='input-field'] ::slotted(:is(input, textarea)) {
      /* Slotted input does not inherit these from the host */
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
```

**The Material Theme:**
```
[part='input-field'] ::slotted(:is(input, textarea)) {
  outline: none;
  margin: 0;
  border: 0;
  border-radius: 0;
  padding: 8px 0;
  width: 100%;
  height: 100%;
  font-family: inherit;
  font-size: 1em;
  line-height: inherit;
  color: inherit;
  background-color: transparent;
  /* Disable default invalid style in Firefox */
  box-shadow: none;
  /* Slotted input does not inherit these from the host */
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
```

Everything is likely fine. Just curious, why the Lumo theme doesn't reset browser styling and actually doesn't apply its own styling for slotted inputs. Does it work somehow differently for the Lumo theme?

Reply:
This was moved to `vaadin-input-container` component `src` styles, but not removed from Material theme.

I can submit a separate PR to cleanup corresponding CSS from Material styles.

Reply:
Ah, understood. 👍  Cleaning the Materials styles would be great. Anyway, it is not an obstacle to merging the PR.